continuousflex/viewers/viewer_pdb_dimred.py

```diff
@@ -90,8 +92,8 @@ def _defineParams(self, form):
                       help='Open a GUI to visualize the PCA space as free energy landscape')
         group.addParam('freeEnergyAxes', StringParam, default="1 2",
                        label='Axes to display' )
-        group.addParam('freeEnergySize', IntParam, default=100,
-                       label='Sampling size' )
+        group.addParam('freeEnergySize', IntParam, default=50,
+                       label='Resolution (pix)' )
```

Comment:
Resolution is already an abused term, maybe we replace it by "window size (pix)"? otherwise, add some description in help